sale_credit_point/tests/test_sale_order.py

```diff
@@ -32,11 +32,15 @@ def _create_so(self):
 
     def test_so_confirm_ok(self):
         so = self._create_so()
-        so.partner_id.credit_point = 100
+        amount = 100
+        comment = 'test_so_confirm_ok'
+        so.partner_id._credit_point_update(amount, comment=comment)
+        so.partner_id.update_history(amount, 'increase', comment=comment)
         self.assertEqual(so.state, 'draft')
         so.action_confirm()
         self.assertNotEqual(so.state, 'draft')
         self.assertEqual(so.partner_id.credit_point, 90)
+        self.assertEqual(so.partner_id.yearly_point_increase, 100)
```

Comment:
why are you testing this here? Unless strictly required, update tests here if needed https://github.com/oleksandrpaziuk/odoo-sale-addons/blob/0163fc5a9299764957effeabe26a47af0e92dbda/sale_credit_point/tests/test_credit_history.py#L57

Reply:
the reason described in the ticket [BSIDM-358](https://jira.camptocamp.com/browse/BSIDM-358)

Reply:
but why are you touching order confirm if you are changing order cancel?

Reply:
Because we should verify that yearly_point_increase don't change when we confirm SO.

Reply:
Due to decoupled (current balance and yearly point limit) nature of sale_credit_point, we can manipulate each part separately but in some moment they need to be synchronized.

Reply:
and how is that possible since we don't add history lines on confirm? You are doing it manually and it's pointless here.
if you want to make sure that 'return' type does not affect yearly credit, test it into test_credit_history

Reply:
Okay!